places-api/ios/mapbox-swiftui/fsq-places-example.xcodeproj/xcuserdata/tsparkman.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>fsq-places-example.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>0</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
```

Comment:
you should `.gitignore` the `xcuserdata` dir and delete the dir from the repo. The current `.gitignore` is missing a lot you should use [this](https://github.com/github/gitignore/blob/main/Swift.gitignore)

Reply:
gitignore updated